signalfx-java/src/main/java/com/signalfx/signalflow/ServerSentEventsTransport.java

```diff
@@ -103,6 +106,34 @@ public Channel execute(String program, final Map<String, String> parameters)
         }
     }
 
+    public Channel preflight(String program, long start, long stop, Long maxDelay)
```

Comment:
It looks like the other methods take an already constructed map of parameters. Follow the pattern?